src/e2e/puppeteer/__tests__/snapshot.ts

```diff
@@ -215,6 +215,28 @@ const testSuite = () => {
       const image = await screenshot()
       expect(image).toMatchImageSnapshot()
     })
+
+    it('drop hover after table', async () => {
+      await paste(`
+        - x
+        - a
+          - =view
+            - Table
+          - =pin
+            - true
+          - b
+            - c
+          - d
+            - e
+      `)
+
+      await clickThought('x')
+      await dragAndDropThought('x', 'd', { position: 'after', mouseUp: true })
```

Comment:
There are a couple problems with this test:

1. If you `mouseUp` here, then this doesn't test the original issue. The issue is about the position of the DropHover, which is shown on drag. If you complete the drop, then the DropHover will no longer be visible. The snapshots you added clearly do not show the expected behavior for the original issue.
2. The code says that the thought is being dragged after d [as a sibling], but the original issue involves dragging after a [below d as an uncle]. This is confusing the semantics of `after` which is meant to be used to drop a thought after a sibling. We need the semantics to align here so that what the test says reflects what is happening in the browser. And we will soon need a test that actually drops `x` after `d` as a sibling anyway.

I think you're going to need a new `position` value to reflect the unique drag-and-drop positioning for tables. #2103 is going to come in helpful here, too.

Reply:
Yes, you are right. The current way the the `after` prop works, does not take into account the children of the target `thought` if it is expanded, since it drags the thought onto the `[x,y] `values of the target thought. We need to implement a new prop that would work in such a case, which includes table view.